app.js

```diff
@@ -122,4 +123,10 @@ function randomItem() {
   console.log(Item.catalog[randomIndex].views);
 }
 
+function updateVotes() {
```

Comment:
As noted in code review, a better name for this method would be something like "summarizeVotes"